curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-ending-string-patterns.md

```diff
@@ -24,7 +24,7 @@ The first `test` call would return `true`, while the second would return `false`
 
 # --instructions--
 
-Use the anchor character (`$`) to match the string `caboose` at the end of the string `caboose`.
```

Comment:
The $ is a RegEx anchor character

Reply:
Okay, So you mean there is no need for change?

Reply:
Here is a section in regex cheat sheet describing it as part of anchors.

https://duckduckgo.com/?q=regex+cheat+sheet&ia=cheatsheet&iax=1

![image](https://user-images.githubusercontent.com/88248797/217728027-7023b2fc-0cd8-4589-b5a9-bb60c3fc6c5e.png)

Reply:
The wording is confusing tho, and I can see it misunderstood by non-native speakers. 

I am happy with rewording it for clearance.

Maybe 🤔

> Use the `$` character, which is part of the regex' anchors, to match the string `caboose` at the end of the string `caboose`.

but I feel it could be better still

Reply:
Yes that's correct, as at the top of the lesson it was referred to as dollar sign, and then at the end it's referred to as anchor sign, so I think its a little confusing.

Reply:
Then there is a need for explaining the regex anchors for students first, for them to know about the topic, and then using the 
`Use the **$** character, which is part of the regex' anchors, to match the string **caboose** at the end of the string **caboose**.`
sentence. Maybe 🤔

Reply:
I could go either way with the change. I think the current isn't wrong wording but the proposed is clearer. We perhaps don't need the vocab term 'anchor' since RegEx is already confusing? 

Just wanted to throw in the context.